common/app/conf/switches.scala

```diff
@@ -471,6 +471,11 @@ object Switches {
     safeState = Off, sellByDate = new LocalDate(2015, 5, 11)
   )
 
+  val ABMtTopBelowNav = Switch("A/B Tests", "ab-mt-top-below-nav",
```

Comment:
It is used in the switchboard eg here https://frontend.code.dev-gutools.co.uk/dev/switchboard